src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -127,7 +127,7 @@ export default function SavedObjectFactory(esAdmin, kbnIndex, Promise, Private,
      * @return {Promise}
      * @resolved {SavedObject}
      */
-    this.init = _.once(() => {
+    this.init = _.once((isDefered) => {
```

Comment:
@spalger  `isDefered`  tells init how to load the saved object. When false, the saved object is fetched immediately. When true, the saved object fetch request is placed on the request queue ... it is deferred. 

The PR was just trying to keeping things simple. I did not want to re-engineer anything. Kibana already had the concept of a request queue and I figured this was a good use for it.